### PR TITLE
allow users to use help and quit before standup has started

### PR DESCRIPTION
**Pull request by JasonMannon**

### CODE REVIEW

File: spec/models/incoming_message_spec.rb [hunk 1]
```diff
@@ -60,10 +60,10 @@
       end
 
       context 'and given the quit command' do
-        let(:text) { '-quit' }
+        let(:text) { '-quit-standup' }
 
         it 'does not create any new messages' do
```
Comment (santiagodoldan) [inline spec/models/incoming_message_spec.rb]: change the description

File: spec/models/incoming_message_spec.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@
         let(:text) { '-help' }
 
         it 'does not create any new messages' do
```
Comment (santiagodoldan) [inline spec/models/incoming_message_spec.rb]: change the description

**Comment by santiagodoldan** [comment]
going to close this one as it was fixed in another commit

**Comment by potherca** [comment]
@santiagodoldan Can you point to which commit that was? this doesn't seem to be working for us. 
(I'd rather investigate further before opening a new issue...)

**Comment by santiagodoldan** [comment]
Seems that we were trying to fix a bug and not allowing those commands before starting the standup session, I think you are right, as it is now it's not possible to execute -quit-standup or -help before entering the -start command, It would be great if you can create a PR for that.

Cheers

**Pull request closed without merging**